### ISSUE TITLE
Intents API: Put extra text doesn't work at DECRYPT_TEXT

### ISSUE THREAD

**Issue by Nequeax** [open-keychain/open-keychain#2230]
## Expected Behavior

Send extra text to new Intent: org.sufficientlysecure.keychain.action.DECRYPT_TEXT.

## Current Behavior

When I add extra text using putExtra("org.sufficientlysecure.keychain.action.EXTRA_TEXT", "Encrypted text") is return answer that: "No data to decrypt"

## Steps to Reproduce (for bugs)

1. Decrypt message using Intents API.
2.
3.
4.

## Your Environment

* Android Version: 
* OpenKeychain Version: 
* From Google Play or F-Droid?:

**Comment by Valodim** [comment]
That's correct, there is no extra "org.sufficientlysecure.keychain.action.EXTRA_TEXT". Where did you get this from, did we incorrectly document it somewhere?

You can use `Intent.ACTION_SEND` as action and `Intent.EXTRA_TEXT` with the data to decrypt text when calling DecryptActivity.

**Comment by Nequeax** [comment]
I found the Intents API documentation here:
https://github.com/open-keychain/open-keychain/wiki/Intents
in section: "OpenKeychain Intent actions"
